gap/perm.gi

```diff
@@ -130,7 +134,13 @@ FindHomMethodsPerm.PcgsForBlocks := function(ri,G)
 end;
 
 #! @BeginChunk BalTreeForBlocks
-#! TODO
+#! This method creates a balanced composition tree for the kernel of an
+#! imprimitive group. This is guaranteed as
+#! <C>FindHomMethodsPerm.BalTreeForBlocks</C> is
+#! just called from <C>FindHomMethodsPerm.Imprimitive</C> and itself.
+#! The homomorphism for the split in the composition tree used is
+#! induced by the action of G on
```

Comment:
Not very important, but: elsewhere you wrote `<A>G</A>`, though I guess one could argue that this is not correct (it is not an argument); another alternative would be `<M>G</M>`. I guess we just should settle on one.